doc/reference/common-weaknesses/centralized-tx.rst

```diff
@@ -0,0 +1,15 @@
+Centralized Transaction Processing Attack
+=========================================
+
+Any centralized transaction processing service represents a single point of failure that could allow for a denial of service attack or simply be overwhelmed during a period of high user load.
+
+Risks
+~~~~~
+
+A selective denial of service attack on a centralized processing resource could allow an attacker to crowd out legitimate users. An example is for an auction contract with a deadline where the attacker secures a winning low bit and then blocks follow on bids by attacking the bid processing server.
+
+Solutions
+~~~~~~~~~
+
+To avoid this kind of vulnerability a contract must be scalable, accepting many inputs per slot—many bids per slot, in the case of an auction. For example, each bid might be made just by creating a UTXO addressed to the auction script, and the auction owner might then collect all of the submitted bids in one transaction—which would permit an arbitrary number of bids to be collected in just two slots.
```

Comment:
This seems confused: the original description talked about a centralized _transaction processing service_ (i.e. off-chain stuff); this is talking about splitting up the _on-chain_ part to allow more to be done in a single slot.

I think there are two issues here:
- DoS attacks on centralized off-chain components
- DoS attacks on on-chain systems with low throughput